src/app/articles/[id]/page.tsx

```diff
@@ -122,7 +126,10 @@ const ArticlePage = (): React.JSX.Element => {
                         </div>
 
                         {/* Action buttons */}
-                        <ProductActions />
+                        <ProductActions
```

Comment:
The props in this component will have to be modified coming front-72 after merging this issue.